tests/functional/mhd_harris_with_boundaries/harris.py

```
@@ -0,0 +1,320 @@
+        boundary_types=("physical", "physical"),
+        boundary_conditions={
+            # x open (free outflow) so tagging puts a refined level on the x physical
+            # boundary, exercising the regrid ghost-fill path; y reflective as before.
+            "xlower": {"type": "open"},
+            "xupper": {"type": "open"},
+            "ylower": {"type": "reflective"},
+            "yupper": {"type": "reflective"},
+        },
+    )
+
+    def S(y, y0, l):
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)